src/electron.js

```diff
@@ -11,6 +11,7 @@ function createWindow() {
       contextIsolation: false,
       enableRemoteModule: false,
     },
+    icon: path.resolve(app.getAppPath(), 'src', 'uc.png'),
```

Comment:
Is it not possible to use the existing icon location?

Reply:
Where is it, I never saw that

Reply:
`build/icon.png`

Reply:
that seems to make it not load.